lib/view_model/dashboard/nft_view_model.dart

```diff
@@ -38,6 +39,8 @@ abstract class NFTViewModelBase with Store {
 
   @action
   Future<void> getNFTAssetByWallet() async {
+    if (appStore.wallet!.type != WalletType.ethereum) return;
```

Comment:
will merge this, but @Blazebrain let's also add polygon when you merge it to Polygon's branch